nilearn/plotting/find_cuts.py

```diff
@@ -76,6 +76,8 @@ def find_xyz_cut_coords(img, mask=None, activation_threshold=None):
         activation_threshold = fast_abs_percentile(my_map[my_map != 0].ravel(),
                                                    80)
     mask = np.abs(my_map) > activation_threshold - 1.e-15
+    if mask.sum() == 0:
```

Comment:
Did you time this with %timeit?

Sent from my phone. Please forgive brevity and mis spelling



On Mar 10, 2015, 12:31, at 12:31, Alexandre Abraham <notifications@github.com> wrote:
>> @@ -76,6 +76,8 @@ def find_xyz_cut_coords(img, mask=None,
>activation_threshold=None):
>>          activation_threshold = fast_abs_percentile(my_map[my_map !=
>0].ravel(),
>>                                                     80)
>>      mask = np.abs(my_map) > activation_threshold - 1.e-15
>> +    if mask.sum() == 0:
>
>Isn't `if not np.any(mask)` slower? In that case, we should also change
>line 73.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/nilearn/nilearn/pull/473/files#r26113990